src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -162,11 +166,10 @@ class JesBackend extends Backend with LazyLogging {
     val jesParameters = standardParameters(callGcsPath) ++ jesInputs ++ unsafeJesOutputs
 
     // Not possible to currently get stdout/stderr so redirect everything and hope the WDL isn't doing that too
-    val redirectedCommand = s"$commandLine > $LocalStdoutValue  2> $LocalStderrValue"
+    val redirectedCommand = s"$instantiatedCommandLine > $LocalStdoutValue 2> $LocalStderrValue"
```

Comment:
It would resolve the "hope the WDL isn't doing that too" part of the comment.

For command line lengths, we have been told the 1024 limit might be raised but that arbitrary length text will not be supported, and furthermore that command scripts are the recommended way to solve this problem.